<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>683</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>League Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="FirstAddButton">
     <property name="text">
      <string>Add</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="FirstEditButton">
     <property name="text">
      <string>Edit</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="FirstDeleteButton">
     <property name="text">
      <string>Delete</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLineEdit" name="LeagueName">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>530</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeagueOID">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>570</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>530</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>League Name:</string>
    </property>
    <property name="buddy">
     <cstring>LeagueName</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>570</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>League OID#:</string>
    </property>
    <property name="buddy">
     <cstring>LeagueOID</cstring>
    </property>
   </widget>
   <widget class="QListWidget" name="LeagueList">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>991</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>530</y>
      <width>60</width>
      <height>64</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="LoadDB">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveDB">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="LoadLeagueButton">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="SaveLeagueDB">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="TeamsNavigatorButton">
   <property name="text">
    <string>To Teams</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
